docs/standard/serialization/snippets/system-text-json-use-dom-utf8jsonreader-utf8jsonwriter/csharp/JsonDocumentWithJsonSerializerOptions.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Text;
+using System.Text.Json;
+using System.Text.Json.Nodes;
+using System.Text.Json.Serialization;
+
+namespace JsonDocumentWithJsonSerializerOptions;
+
+public class Program
+{
+    public static void Main()
+    {
+        Person person = new Person { Name = "Nancy" };
+
+        // Serialize and ignore null properties - null Address property is omitted
+        // Output: {"Name":"Nancy"}
+        JsonSerializerOptions options = new()
+        {
+            DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingNull
+        };
+        string personJson = JsonSerializer.Serialize(person, options);
+        Console.WriteLine(personJson);
+
+        // Default serialization - Address property included with null token.
```

Comment:
Would it make sense to show the default behavior first?

Reply:
Sorry, missed this comment earlier. Yes it does, I'll change it.